browser/resources/settings/brave_overrides/settings_menu.ts

```diff
@@ -297,29 +297,15 @@ RegisterPolymerTemplateModifications({
     )
     searchEl.insertAdjacentElement('afterend', extensionEl)
 
-    const isBraveWalletAllowed = loadTimeData.getBoolean('isBraveWalletAllowed')
-    let walletEl = undefined
-    if (isBraveWalletAllowed) {
-      walletEl = createMenuElement(
-        loadTimeData.getString('braveWallet'),
-        '/wallet',
-        'brave_settings:wallet',
-        'wallet',
-      )
-      extensionEl.insertAdjacentElement('afterend', walletEl)
-    }
 
-    const ipfsEl = createMenuElement(
-      loadTimeData.getString('braveIPFS'),
-      '/ipfs',
-      'brave_settings:ipfs',
```

Comment:
I think this means we should remove the ipfs icon from brave_settings unless we show it anywhere else?

Reply:
https://github.com/brave/brave-core/pull/16290/commits/46486a1a3ed9dde0cb211de0cc787d5d5b15ac86